apache/couchdb-fauxton
# Remove JQuery uses from documents addon

## Overview

Remove JQuery uses from documents addon.
Also as I was testing cancellation of a file upload, I noticed the 'Upload' button and 'Choose file' were still active when the upload is in progress. I fixed that so both will be disabled while upload is in progress.

## Testing recommendations

Test these features:
 - Bulk delete docs
 - Upload attachment to existing doc
   - Cancel an upload
   - While upload is in progress, user should only be able to press Cancel / Close the modal.
 - Delete document from Document Editor
 - Create new doc and check that `_id` field is set to a random UUID value
 - Go to All Docs page, then check that the Search combobox at the top works
 - List database changes

To test cancelling a file upload, it helps to throttle the network speed using in the Network tab of Chrome's devtools.

## Checklist
